src/components/Modals/AddContactModal.jsx

```diff
@@ -12,49 +12,43 @@ import FormControl from '@mui/material/FormControl';
 import IconButton from '@mui/material/IconButton';
 import InputAdornment from '@mui/material/InputAdornment';
 import TextField from '@mui/material/TextField';
-// Utility Imports
 import { runNotification } from '@utils';
-// Context Imports
-import { UserListContext } from '@contexts';
-// Model Imports
-import { createUser } from '../../model-helpers/User';
-// Component Imports
 import { FormSection } from '../Form';
 
 /**
- * AddClientModal Component - Component that allows users to add other user's
- * Pod URLs from a user's list stored on their own Pod
- *
- * @memberof Modals
- * @name AddClientModal
+ * @memberof Contcts
+ * @name renderWebId
+ * @param {string} username - username to convert into a webId
+ * @returns {URL} A url of the predicted webID
  */
-
 const renderWebId = (username) => {
   const baseUrl = new URL(localStorage.getItem('oidcIssuer'));
   return new URL(`${username}/profile/card#me`, baseUrl);
 };
 
-const AddClientModal = ({ showAddClientModal, setShowAddClientModal }) => {
+/**
+ * AddContactModal Component - Component that allows users to add other user's
+ * Pod URLs from a user's list stored on their own Pod
+ *
+ * @memberof Contacts
+ * @name AddContactModal
+ * @param {Function} addContact  - function to add a contact
```

Comment:
The param for the component should be a React Prop. I believe you've implemented them correctly in the components in the new `src/components/Contacts`

Reply:
fixed